README.md

```diff
@@ -94,15 +116,25 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+For function();
 
+calling would be inputting the foillowing: function();
+
+referencing would be simply inputting "function;"
 ```
 ### Question #9
 
 Using the object literal notation, Define an object called student and give it the properties (your attributes) of name, age, and a method sayHello, that outputs "Hi, my name is [your_name]".
 
 Your Answer:
-```js
-// write code here
+```js 
+var student= {
+  name: "Vijay",
+  age: 28,
+  sayHello: function(){
+    console.log("Hi, my name is" + " " + "student.name");
+  }
+}
```

Comment:
Almost! This would print out "Hi, my name is student.name.". In order for `student.name` to behave as a variable, it should *not* be in quotes.

Also, no need to have that space in quotes by itself.